purchase_analytic_tag_dimension/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
add ending blankline and header with at least the coding `# -*- coding: utf-8 -*-`

Reply:
Here there is no code, common sense is that we don't need copyright reminder in this file.

Reply:
I'm not talking about the copyrights, I refer to coding and even license as [OCA template](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py) suggest.